.coveragerc

```diff
@@ -1,2 +1,3 @@
 [run]
-omit = /tmp/*
+omit = /tmp/*[run]
```

Comment:
What does that pattern mean? Just a copying mistake? If it follows typical globbing rules, then this would only omit files starting with `/tmp/` and ending with `r`, `u`, or `n`

Reply:
@chmeliik; :facepalm: